client/blocks/continue-as-user/index.jsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default function ContinueAsUser( { userName, redirectUrl } ) {
+	const translate = useTranslate();
+
+	if ( ! userName || ! redirectUrl ) {
+		return null;
+	}
+
+	const redirectLink = <a href={ redirectUrl }>{ userName }</a>;
+
+	return (
+		<div className="continue-as-user">
+			{ translate( 'or continue as {{redirectLink/}}', {
```

Comment:
Would `continue as {{userName/}}` better convey the meaning to translators?

Reply:
Yes, very likely! Fixed.